CryptoPrices.xcodeproj/project.pbxproj

```diff
@@ -553,7 +575,7 @@
 				);
 				MARKETING_VERSION = 0.1.0;
 				PRODUCT_BUNDLE_IDENTIFIER = "co.nimblehq.crypto-prices.staging";
-				PRODUCT_NAME = "$(TARGET_NAME) Staging";
+				PRODUCT_NAME = "$(TARGET_NAME)";
```

Comment:
Why do we need this change? As shouldn't we provide different product names for different schemes? 🤔

Reply:
At first, I encountered an error related to TEST_HOST, which led me to make a workaround to fix it by making PRODUCT_NAME unique. But actually, that's not a good workaround you mentioned above 😓 .

![Screenshot 2022-12-02 at 14 11 13](https://user-images.githubusercontent.com/25881847/205248868-90157b9d-3dda-49f3-a240-a96adefc900d.png)

Finally, I fixed it by reverting the PRODUCT_NAME values with the `Staging` suffix, making a unique PRODUCT_MODULE_NAME's name, and correcting the path of TEST_HOST for each scheme at https://github.com/nimblehq/nimble-crypto-ios/pull/23/commits/7b42f77489e194e433da15dbe23d01a419cbb87d.

Thank you so much for raising it :D.